napari/layers/labels/labels.py

```diff
@@ -395,30 +402,43 @@ def data(self, data):
         self.events.data(value=self.data)
         self._set_editable()
 
+    @property
+    def features(self):
+        """DataFrame-like features table.
+
+        Treat this as a very simple pandas DataFrame that maps a feature name
```

Comment:
Is there a little more detail that can be provided here? maybe a link where folks can learn more? i can see folks being confused reading this

Reply:
I like the wording above and the idea of adding the public utility function that we can reference for people that want to guarantee a pandas DataFrame.

Reply:
How about the following?

```
    @property
    def features(self):
        """ Dataframe-like features table.

        It is an implementation detail that this is a `pandas.DataFrame`. In the future,
        we will target the currently-in-development Data API dataframe protocol [1].
        This will enable us to use alternate libraries such as xarray or cuDF for
        additional features without breaking existing usage of this.

        If you need to specifically rely on the pandas API, please coerce this to a
        `pandas.DataFrame` using `features_to_pandas_dataframe`.

        References
        ----------
        .. [1]: https://data-apis.org/dataframe-protocol/latest/API.html
        """
```

My only concern is that this does not tell the user what they can do with this, only what they might be able to do in the future.

Reply:
We decided that this docstring should suffice for now. We'll try to improve as needed and as people start to use features.